climate_emotions_map/app.py

```diff
@@ -19,5 +20,41 @@
 
 server = app.server
 
+
+@callback(
+    Output("drawer", "opened"),
+    Input("drawer-button", "n_clicks"),
+    prevent_initial_call=True,
+)
+def drawer_demo(n_clicks):
+    """Callback function for toggling drawer visibility."""
+    return True
+
+
+@callback(
+    Output("drawer-state", "children"),
+    Input("state-select", "value"),
+)
+def update_drawer_state(value):
+    """Callback function for updating the state in the drawer."""
+    if value is None:
+        return "National"
+    else:
```

Comment:
I don't think you need the `else` here, since the previous `if` returns you out of the function!